src/bitcoin/d/mod.rs

```diff
@@ -1060,11 +1065,18 @@ impl From<Json> for GetTxRes {
                     })
                     .collect()
             });
-
+        let hex = json
+            .get("hex")
+            .and_then(Json::as_str)
+            .expect("Must be present in bitcoind response");
+        let bytes = Vec::from_hex(hex).expect("bitcoind returned a wrong transaction format");
+        let tx: bitcoin::Transaction = bitcoin::consensus::encode::deserialize(&bytes)
+            .expect("bitcoind returned a wrong transaction format");
```

Comment:
nit: i think you can just use from_str. Something like
```suggestion
        let hex = json
            .get("hex")
            .and_then(Json::as_str)
            .and_then(|hex| bitcoin::Transaction::from_str(&hex).ok())
            .expect("A valid hex transaction must always be present");
```